### DISCUSSION TITLE
想对接钱包的功能

### DISCUSSION THREAD

**Discussion by xiaoyuzaixian2013** [consenlabs/webview#34] [Q&A] [answered]
收款只收usdt，只用这一种方式进行收款，客户付钱后直接到钱包，有没有成套完整的开发文档呢？有的话麻烦发我一下，谢谢
而且开发者账号我也已经申请开通了，但是没有系统的文档，不知道如何对接呢？一般都是有接口或者文档一类的，才好进行开发

**Comment by unix** [comment, marked as answer]
imToken 是去中心化的数字钱包，当前并不提供第三方的支付、收款集成。上文提到的类似需求无法使用 imToken 相关 SDK 完成。
